src/components/uploader/js/Uploader.js

```diff
@@ -200,20 +191,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.uploader.updateTotalProgress = function (that) {
-        // Preservered for backwards compatibility, should be refactored post v1.5
         var batch = that.queue.currentBatch;
         var totalPercent = fluid.uploader.derivePercent(batch.totalBytesUploaded, batch.totalBytes);
-        var numFilesInBatch = batch.files.length;
-        var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
-
         var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
 
         var totalProgressStr = fluid.stringTemplate(that.options.strings.progress.totalProgressLabel, {
-            curFileN: batch.fileIdx,
-            totalFilesN: numFilesInBatch,
-            fileLabel: fileLabelStr,
-            currBytes: fluid.uploader.formatFileSize(batch.totalBytesUploaded),
```

Comment:
I'm concerned by these changes which appear to be removing functionality from the uploader as well as compatibility information. I'm also concerned that this information could be removed without breaking any tests...